uwblueprint/sistering
# Update 404 page

## Ticket link

Closes #600



## Implementation description
* Updated the 404 page to match the new design on Figma



## Steps to test
1.Navigate to any invalid url, verify the following 404 is displayed 
<img width="758" alt="Screen Shot 2022-08-15 at 5 30 20 PM" src="https://user-images.githubusercontent.com/82241706/184759337-4e2800a3-e6ed-479e-a5ea-2b50129eead9.png">




## What should reviewers focus on?
* Verify the implementation matches the figma (figma shown below)
<img width="735" alt="Screen Shot 2022-08-15 at 5 31 21 PM" src="https://user-images.githubusercontent.com/82241706/184759365-27d943f4-3132-4dde-9e9d-b3f5c4884074.png">



## Checklist
